turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/dev/runtime/base/runtime-base.ts

```diff
@@ -402,6 +406,12 @@ function resolveAbsolutePath(modulePath?: string): string {
   return `/ROOT/${modulePath ?? ""}`;
 }
 
+function getWorkerBlobURL(chunks: ChunkPath[]): string {
+  let bootstrap = `TURBOPACK_WORKER_LOCATION = "${location.origin}";importScripts(${chunks.map(c => (`TURBOPACK_WORKER_LOCATION + "${getChunkRelativeUrl(c)}"`)).join(", ")});`;
```

Comment:
This global variable is used to forward the page's origin to the worker (blob), so that the non-relative blob origin can load the scripts.